.travis.yml

```diff
@@ -13,6 +13,7 @@ env:
   - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
 addons:
   postgresql: '10'
+before_install: sudo apt-get install -y libqpid-proton2 libqpid-proton2-dev
```

Comment:
@aufi we should use the same qpid-proton version as the nuage provider (0.30.0), the version in xenial looks like 0.10.0.

We use 0.30.0 because that is the version of the C library that we can install on the appliance, see https://github.com/ManageIQ/manageiq-providers-nuage/blob/master/tools/ci/before_install.sh and https://github.com/ManageIQ/manageiq-providers-nuage/blob/master/.travis.yml#L27

Reply:
Thanks, updated!